github-templates/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+**Feature Request** or **Bug Report**
```

Comment:
Definitely in favour of that. I'll update the issue templates with a blurb here to tag new issues with the appropriate label.